runbotics/runbotics-scheduler/src/mail/mail.service.ts

```diff
@@ -53,18 +60,62 @@ export class MailService {
     }
 
     public async sendBotDisconnectionNotificationMail(bot: IBot, installationId: string) {
-        await this.sendMail({
-            to: bot.user.email ?? '',
-            subject: NOTIFICATION_MAIL_SUBJECT,
-            content: `Hello,\n\nBot 🤖 (${installationId}) has been disconnected.\n\nBest regards,\nRunBotics`,
-        });
+        const disconnectedBot = await this.botService.findById(bot.id);
+
+        if (disconnectedBot.subscribers && disconnectedBot.subscribers.length) {
+            const subscribers = disconnectedBot.subscribers;
+            const emailAddresses = subscribers.map(({ email }) => email).join(',');
+
+            await this.sendMail({
+                to: emailAddresses,
+                subject: NOTIFICATION_MAIL_SUBJECT,
+                content: `Hello,\n\nBot 🤖 (${installationId}) has been disconnected.\n\nBest regards,\nRunBotics`,
```

Comment:
imo lets include info about app instance. Include link to the instance, should be available in RUNBOTICS_ENTRYPOINT